app/models/station.rb

```diff
@@ -2,4 +2,5 @@
 
 class Station < ApplicationRecord
   has_many :favorites
+  has_many :users, through: :favorites
```

Comment:
didn't know why this wasn't reflected on both sides, so put it back

Reply:
The reasoning was that a station doesnt have ownership over a user, so it wasnt really doing anything

Reply:
😼

Reply:
VBD format aside, it doesnt hurt and now given a station object you could theoretically get all of the stations they liked. valuable data, i need to know what ur fav station is